docs/source/index.rst

```diff
@@ -15,6 +15,9 @@ This documentation will guide you through the installation, deployment, and laun
 
 As Hyperledger Iroha is an open-source project, we will also cover contribution part and explain you a working process.
 
+.. note::
+    Please note that there exists a complete rewrite of Hyperledger Iroha in Rust, `Iroha 2 <https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha>`_. Both projects are developed concurrently in the same repository. However, they are not compatible. You can read about their differences in `Iroha 2 documentation <https://hyperledger.github.io/iroha-2-docs/guide/iroha-2.html>`_.
```

Comment:
```suggestion
    Please note  `Iroha 2 <https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha>`_ is recommended for all new projects, instead of Iroha 1. Both projects are developed concurrently in the same repository. However, they are not compatible. You can read about their differences in `Iroha 2 documentation <https://hyperledger.github.io/iroha-2-docs/guide/iroha-2.html>`_.
```

Reply:
```suggestion
    Please note that there exists a complete rewrite of Hyperledger Iroha in Rust, `Iroha 2 <https://github.com/hyperledger/iroha/tree/iroha2-dev#hyperledger-iroha>`_. Both projects are developed concurrently in the same repository. However, they are not compatible. You can read about their differences in `Iroha 2 documentation <https://hyperledger.github.io/iroha-2-docs/guide/iroha-2.html>`_.
    
    For all new projects, we recommend using Iroha 2 instead of Iroha 1.
```